packages/airnode-validator/src/ois/ois.ts

```diff
@@ -106,11 +113,50 @@ export const operationSchema = z.object({
   parameters: z.array(operationParameterSchema),
 });
 
-export const pathSchema = z.record(operationSchema);
+export const httpStatusCodes = z.union([z.literal('get'), z.literal('post')]);
+
+export const pathSchema = z.record(httpStatusCodes, operationSchema);
+
+const ensurePathParametersExist: ValidatorRefinement<SchemaType<typeof pathsSchema>> = (paths, ctx) => {
+  forEach(paths, (pathData, rawPath) => {
+    forEach(pathData, (paramData, httpMethod) => {
+      const parameters = paramData!.parameters;
+      // Match on anything in the path that is braces
+      // i.e. The path /users/{id}/{action} will match ['{id}', '{action}']
```

Comment:
These comments are very helpful 👍🏻

Reply:
Hehe :) , I copied this over from https://github.com/api3dao/airnode/blob/master/packages/airnode-adapter/src/request-building/path-parser.ts#L12 so they are most likely yours :D